wshaman/elastic-integrations
# Adding New Test Cases

## Proposed commit message

 This pull request introduces  new test cases.The additions have been made to two test files: "test-inventory.json" and "test-inventory.json-expected.json".

## Checklist


- [x] Verify that expected test cases have been added to test-inventory.json-expected.json.
- [x] Ensure that new test cases have been incorporated into test-inventory.json.
- [x] Changes added to fields.yml.


## How to test this PR locally

 To verify the changes introduced in this pull request run the following command to execute the pipeline tests: elastic-package test pipeline.
